Source/Engine/Level/Tags.cpp

```diff
@@ -55,6 +55,23 @@ Tag Tags::Get(const StringView& tagName)
     return tag;
 }
 
+Array<Tag> Tags::GetSubTags(Tag parentTag)
+{
+    Array<Tag> subTags;
+    auto _parentTagName = parentTag.ToString();
+
+    for (int i = 0; i < Tags::List.Count(); i++)
+    {
+        auto& subTagName = Tags::List[i];
+        if (subTagName.Contains(_parentTagName) && subTagName != _parentTagName)
+        {
+            subTags.Add(Tags::Get(subTagName));
```

Comment:
`Tag(i + 1)`instead of `Tags::Get(subTagName)` as it's already in that tags list